### PR TITLE
Added function body and tests for ggpair plot

### PR DESCRIPTION
**Pull request by Jaskaran1116**

### CODE REVIEW

File: R/correlation_graph.R [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#' scatter2d 
+#' A short-cut function for creating 2 dimensional ggpair scatterplots via GGally and ggplot2
+#'
+#' @param data data.frame or tibble
+#' @param title title of the plot from the data frame or tibble 
+#'
+#' @return
```
Comment (a-kong) [inline R/correlation_graph.R:7]: @return should specify what the function returns (a ggpair scatterplot)

File: R/correlation_graph.R [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#' scatter2d 
+#' A short-cut function for creating 2 dimensional ggpair scatterplots via GGally and ggplot2
+#'
+#' @param data data.frame or tibble
```
Comment (a-kong) [inline R/correlation_graph.R:4]:
```suggestion
#' @param data A data frame or tibble.
```

File: R/correlation_graph.R [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#' @export
+#'
+#' @examples
+#' scatter2d(mtcars, hp, mpg)
+
+correlation_graph <- function(.data, gg_title) {
+  GGally::ggpairs(data = .data, 
+  ggplot2::aes(colour = "steelblue", alpha = 0.5)) + 
+    ggplot2::ggtitle(gg_title)
+}
```
Comment (a-kong) [inline R/correlation_graph.R:17]: after styling:
```suggestion
correlation_graph <- function(.data, gg_title) {
  GGally::ggpairs(data = .data, 
                  ggplot2::aes(colour = "steelblue", alpha = 0.5)) + 
    ggplot2::ggtitle(gg_title)
```

File: R/correlation_graph.R (hunk 1 above)
Comment (samzzzzzh) [reply R/correlation_graph.R:7]: Good catch. The specification should indicate the output.

**Review by samzzzzzh — APPROVED** [review]
I think everything looks great! Just some small syntax issues but code looks solid! Good job.

**Review by a-kong — APPROVED** [review]
just a few changes, but looks good!

**Pull request merged**